### PR TITLE
we moved the security file to fluxcd/.github

### PR DESCRIPTION
**Pull request by dholbach**
New location is here: https://github.com/fluxcd/.github/blob/main/SECURITY.md

### CODE REVIEW

File: external-sources/fluxcd/community [hunk 1]
```diff
@@ -2,4 +2,3 @@
 "/SUPPORT.md","/support.md"
 "/CONTRIBUTING.md","/contributing.md"
 "/GOVERNANCE.md","/governance.md"
-"/SECURITY.md","/security.md"
```
Comment (stefanprodan) [inline external-sources/fluxcd/community:5]: New line here please

**Comment by stefanprodan** [comment]
First we need to remove the link from the document, no idea why SECURITY.md links to itself but it shouldn't.

**Comment by dholbach** [comment]
> First we need to remove the link from the document, no idea why SECURITY.md links to itself but it shouldn't.

I could imagine it was originally placed there so if the file ever got copied around it was clear where the main copy was. I'm happy to remove the link, or point to its new location. WDYT?

**Comment by stefanprodan** [comment]
@dholbach I looked at other CNCF projects, no one has such a link, for example: https://github.com/coredns/coredns/security/policy

**Comment by dholbach** [comment]
> @dholbach I looked at other CNCF projects, no one has such a link, for example: https://github.com/coredns/coredns/security/policy

https://github.com/fluxcd/.github/pull/2

**Review by stefanprodan — APPROVED** [review]

**Pull request merged**